examples/bri-3/prisma/migrations/20230821213120_add_bpi_account_state_tree_storage/migration.sql

```diff
@@ -0,0 +1,29 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `stateObjectStorage` on the `BpiAccount` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "BpiAccount" DROP COLUMN "stateObjectStorage";
```

Comment:
there is already a migration on main that executes this

Reply:
removed